src/app/components/core/control-dispatcher/api-flow/api-flow.component.ts

```diff
@@ -0,0 +1,246 @@
+import { Component, OnInit } from '@angular/core';
+import { ControlAPIFlowService, ProgressBarService, RolesService, SharedService, AlertService } from '../../../../services';
+import { ControlDispatcherService } from '../../../../services/control-dispatcher.service';
+import { DocService } from '../../../../services/doc.service';
+import { DialogService } from '../../../common/confirmation-dialog/dialog.service';
+import { QUOTATION_VALIDATION_PATTERN } from '../../../../utils';
+import { NgForm, Validators, FormGroup, FormBuilder, AbstractControl, FormArray } from '@angular/forms';
+import { UserService } from '../../../../services';
+
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+
+import { APIFlow, User } from '../../../../../../src/app/models';
+
+@Component({
+    selector: 'app-api-flow',
+    templateUrl: './api-flow.component.html',
+    styleUrls: ['./api-flow.component.css']
+})
+
+export class APIFlowComponent implements OnInit {
+    apiFlows = []; // TODO: typecast APIFlow
+    epName: string = '';
+    allUsers: User[];
```

Comment:
Add a line break before this, to logically group the member variables & better readability.